source/guide_rustdesk.rst

```diff
@@ -0,0 +1,178 @@
+
+RustDesk_ is an open-source remote desktop software that allows remote access to and control of other computers. In addition to a client, which is available for various operating systems, a server compound is also required for `hole punching`_ (see: `How does RustDesk work?`_). Although there are public servers, it is possible to host a RustDesk server yourself, as there is a free version (`RustDesk Server`_) in addition to a paid version (RustDesk Server Pro).
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Firewall ports <basics-ports>`
+  * :manual:`supervisord <daemons-supervisord>`
+
+License
+=======
+
+All relevant legal information can be found here:
+
+  * https://github.com/rustdesk/rustdesk-server/blob/master/LICENSE
+
+Prerequisites
+=============
+
+To make the application accessible from the outside, open three `ports in the firewall <firewall_>`_:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ uberspace port add
+  Port 40130 will be open for TCP and UDP traffic in a few minutes.
+  [isabell@stardust ~]$ uberspace port add
+  Port 40131 will be open for TCP and UDP traffic in a few minutes.
+  [isabell@stardust ~]$ uberspace port add
+  Port 40132 will be open for TCP and UDP traffic in a few minutes.
+  [isabell@stardust ~]$
+
+
+At least two of these should be consecutive, as one of them cannot be specified explicitly, but results from the other selected port. In our case, we will specify ``40131`` and ``40132`` explicitly later (see :ref:`setup-daemon`), ``40130`` will then be used automatically.
+
+Installation
+============
+
+Create a new directory, enter the directory you just created, download the latest version, unpack the archive and delete it afterwards:
+
+.. warning:: Replace ``1.1.10-3`` with the version of the latest release!
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mkdir ~/rustdesk
+ [isabell@stardust ~]$ cd ~/rustdesk
+ [isabell@stardust rustdesk]$ wget "https://github.com/rustdesk/rustdesk-server/releases/download/1.1.10-3/rustdesk-server-linux-amd64.zip"
+ [isabell@stardust rustdesk]$ unzip -j rustdesk-server-linux-amd64.zip
+ [isabell@stardust rustdesk]$ rm rustdesk-server-linux-amd64.zip
+ [isabell@stardust rustdesk]$
+
+Configuration
+=============
+
+Configure Webserver
+-------------------
+
+.. _setup-daemon:
+             
+Setup daemon
+------------
+
+.. warning:: Replace ``isabell`` with your username and ``40131``/``40132`` with your ports!
+
+Create ``~/etc/services.d/rustdesk_hbbs.ini`` with the following content:
+
+::
+
+ [program:rustdesk_hbbs]
+ directory=%(ENV_HOME)s/rustdesk/
+ command=%(ENV_HOME)s/rustdesk/hbbs -r isabell.uber.space:40132 -p 40131 -k _
+ autostart=yes
+ autorestart=yes
+
+Create ``~/etc/services.d/rustdesk_hbbr.ini`` with the following content:
+
+::
+
+ [program:rustdesk_hbbr]
+ directory=%(ENV_HOME)s/rustdesk/
+ command=%(ENV_HOME)s/rustdesk/hbbr -p 40132 -k _
+ autostart=yes
+ autorestart=yes
+
```

Comment:
To avoid unnecessary load due to restart loops in case of an error, 'startsecs=30' would also be useful. (On both services)

Reply:
also done!